web/src/compose_actions.js

```diff
@@ -492,6 +492,7 @@ export function quote_and_reply(opts) {
 
     channel.get({
         url: "/json/messages/" + message_id,
+        data: {stream_wildcard_mentioned_flag: true},
```

Comment:
Hmm, I don't love the idea that we need to remember to pass this parameter in every endpoint fetching messages.

Reply:
With the [updated approach,](https://chat.zulip.org/#narrow/stream/378-api-design/topic/highlight.20topic-wildcard.20mention/near/1656239) we don't need to pass these params.